content/trusted-content/official-images/using.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Using Docker Official Images
+description: |
+  Learn about building applications with Docker Official images
+  and how to interpret the tag names they use.
+keywords: docker official images, doi, tags, slim, feedback, troubleshooting
+weight: 10
+---
+
+Docker recommends you use the Docker Official Images in your projects.
+These images have clear documentation, promote best practices, and are regularly updated.
+Docker Official Images support most common use cases, making them perfect for new Docker users.
+Advanced users can benefit from more specialized image variants as well as review Docker Official Images as part of your `Dockerfile` learning process.
+
+## Tags
+
+The repository description for each Docker Official Image contains a
+**Supported tags and respective Dockerfile links** section that lists all the
+current tags with links to the Dockerfiles that created the image with those
+tags. The purpose of this section is to show what image variants are available.
+
+![Example: supported tags for Ubuntu](../images/supported_tags.webp)
```

Comment:
Given this is a screenshot of plain markdown as rendered by Hub, would it be cleaner to just embed that same markdown in a quoted section here instead of adding a binary blob of an image (and in a file format that GitHub doesn't natively even recognize as an image, so it doesn't show up in diffs)?  That would make it easier to update over time if you wanted or needed to:

```suggestion
> # Supported tags and respective `Dockerfile` links
>
> -	[`20.04`, `focal-20240216`, `focal`](https://git.launchpad.net/cloud-images/+oci/ubuntu-base/tree/oci/index.json?h=refs/tags/dist-focal-amd64-20240216-5e82aa8e&id=5e82aa8e3498567df6757d3e9598a8cd54d19ee9)
> -	[`22.04`, `jammy-20240227`, `jammy`, `latest`](https://git.launchpad.net/cloud-images/+oci/ubuntu-base/tree/oci/index.json?h=refs/tags/dist-jammy-amd64-20240227-c988e436&id=c988e4364bc3cc0378f6c3592eb486d281a6ac41)
> -	[`23.10`, `mantic-20240216`, `mantic`, `rolling`](https://git.launchpad.net/cloud-images/+oci/ubuntu-base/tree/oci/index.json?h=refs/tags/dist-mantic-amd64-20240216-c9ad3939&id=c9ad39394a666b231d5c5398e3cd38ac70c0594c)
> -	[`24.04`, `noble-20240225`, `noble`, `devel`](https://git.launchpad.net/cloud-images/+oci/ubuntu-base/tree/oci/index.json?h=refs/tags/dist-noble-amd64-20240225-8cf671e3&id=8cf671e3c71fa7d0735ceefd4a77c7af2771472f)
```

Reply:
I think the screenshot adds an extra touch of familiarity if you've come across that readme section on Docker Hub, but not quite understood it. I'd prefer to keep the screenshot for that reason I think